deneme3.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+from py2neo import authenticate, Graph,Node
+
+import argparse
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--user', type=str, default="neo4j", help='username for neo4j')
+parser.add_argument('--password', type=str, default="777", help='password of the user')
+
+args = parser.parse_args()
+
+authenticate("localhost:7474", args.user, args.password)

```

Comment:
Bu şekilde bağlantı sebebiyle (ya da memory'nin yetmeyebileceği durumlarda mesela) arada hata alabiliriz. Bunlar için exception yazalım. (Search key: Python errors and exceptions)